test/unit/gateways/global_collect_test.rb

```diff
@@ -9,6 +9,27 @@ def setup
                                         secret_api_key: '109H/288H*50Y18W4/0G8571F245KA=')
 
     @credit_card = credit_card('4567350000427977')
+    @apple_pay_network_token = network_tokenization_credit_card('4444333322221111',
+      month: 10,
+      year: 24,
```

Comment:
Unless there's something different about ApplePay test tokens, might be better to also make this dynamic